mesheryctl/internal/cli/root/app/app.go

```diff
@@ -69,20 +69,17 @@ func init() {
 func getSourceTypes() error {
 	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
 	if err != nil {
-		utils.Log.Error(err)
-		return nil
+		return utils.ErrLoadConfig(err)
 	}
 	validTypesURL := mctlCfg.GetBaseMesheryURL() + "/api/application/types"
 	req, err := utils.NewRequest("GET", validTypesURL, nil)
 	if err != nil {
-		utils.Log.Error(err)
-		return nil
+		return utils.ErrCreatingRequest(err)
```

Comment:
Same as above i have already refactored app and pattern package  please work on other packages